interface1.html

```diff
@@ -1057,6 +1058,10 @@ <h2>Mapping constraints
 								<div class="category-selection-div"><input type="checkbox" name="category-matchCustomSP" value="cp">CP</div>
 								<div class="category-selection-div"><input type="checkbox" name="category-matchCustomSP" value="xp" checked="checked">XP</div>
 								<div class="category-selection-div"><input type="checkbox" name="category-matchCustomSP" value="x0">X<sup>0</sup></div>
+								<div style="white-space:nowrap" class="category-selection-div"><input type="checkbox" name="category-matchCustomSP" value="any"><div display="inline-block">Ignore Prosodic Categories</div></div>
```

Comment:
Line break problem here, at least in Chrome on MacOS.

Reply:
Don't capitalize Prosodic Categories.

Reply:
Doesn't look like this is working as labeled. It's set up to do any syntactic category, instead of matching the specified syntactic category to any prosodic category.

Let's talk about how to set up these options on the interface because I'm realizing there are more options for how to do this than i had initially realized.

Reply:
Further clarification for this one. Is this the one? 
On interface: putting it above MatchNonminimal with the name:  Match(Syntax -> Prosody), any prosodic constituent
Options: 
[ ] cp [ ] xp [ ] x0
[ ] all syntactic categories

On the backend, call: matchSP(stree, ptree, cat, {anyPCat: true})

Just want to clarify before adding anything. Thank you!

Reply:
I'm not sure I've understood the question perfectly. That comment is the proposal I want you to implement, and this is one place in the interface code where a change will need to happen. I hope that addresses your question!

Reply:
ok, that helps.

Reply:
Yep, I was working on an answer there and got caught up addressing a bug that I stumbled across (#581 ). I just finished replying to your question on #450.

Reply:
I looked through the files but I can't figure out a way to make the backend call. It works like the other match options when there is no input but when there is an input, there is a globalNameOrDirect error. Could you direct me to the file where the backend call for the other match options is happening? Is it in the built-in_analyses.js in the record_analysis() function? Thanks!

Reply:
globalNameOrDirect means the constraint function name isn't being recognized. Handling of options on the interface happens in main/interface/interface_tableau.js in the getCheckedConstraints function. But another route you can take is to copy the way that MatchMax is handled on the interface -- make a function matchSPAny() that just calls matchSP() with {anyPCat:true} as the options object. Look at the function matchMaxSP() in main/constraints/match.js to see how this works.

Reply:
I tried the way with the matchSPAny() but I got a globalNameOrDirect due to it. I am not sure if it is due to my naming and changed it to matchSPAny() but that still didn't work. We could discuss this during the meeting if that works.